src/views/HomeView.jsx

```diff
@@ -324,6 +324,18 @@ function HomeView() {
         </Card>
       </div>
 
+      <div style={styles.cardBoundary}>
+        <Card style={styles.cardInterior}>
+          <Card.Title style={styles.cardTitle}>System Status</Card.Title>
+          <Card.Text>
+            <b>Product</b>: Buttons and Sensor
+            <hr />
+            The System Status tool summarizes the health of our Brave Devices. Depending on the environment selected in the nav bar, this tool checks
```

Comment:
Good to mention here that the environment is dependent on what is currently selected in the navigation bar.